lib/import-notebook.js

```diff
@@ -77,18 +113,34 @@ export async function _loadNotebook(filename: string) {
   const editor = await atom.workspace.open();
   const grammar = getGrammarForNotebook(nb);
   if (!grammar) return;
-  editor.setGrammar(grammar);
+  atom.grammars.assignLanguageMode(editor, grammar.scopeName);
   const commentStartString = getCommentStartString(editor);
   if (!commentStartString) {
     atom.notifications.addError("No comment symbol defined in root scope");
     return;
   }
-  const sources = _.map(nb.cells, cell => {
-    return getCellSource(cell, commentStartString + " ");
-  });
+  const nbCells = nb.cellMap.valueSeq().toArray();
+  let sources = [];
+  let resultRows = [];
+  let previousBreakpoint = -1;
+  for (let cell of nbCells) {
```

Comment:
I think we usually use `forEach` for for loops without break/return statements inside, thus I believe it's more preferable here too.

If you stick to use native `for`, I prefer `const` over `let` for `cell` 's identifier